src/containers/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+import LoginContainer from '../../containers/Login';
+import NavBar from '../../components/landingPage/NavBar';
+import Header from '../../components/landingPage/Header';
+import Carousel from '../../components/landingPage/Carousel';
+import ArticleCardContainer from './ArticleCardContainer';
+import Footer from '../../components/landingPage/Footer';
+
+import '../../styles/landing.scss'
+
+class LandingPage extends Component {
+  state = {
+    open : false
+  }
+
+  onOpenModal = () => {
+    this.setState({ open: true });
+  }
+
+  onCloseModal = () => {
+    this.setState({ open: false });
+  };
+  
+  buttonClicked = (e) =>{
+    e.preventDefault;
```

Comment:
@walimike , use 'event' instead of 'e'. We don't know what 'e' is?

Reply:
Done